야리무의 쇼핑몰/styles.css

```
@@ -0,0 +1,90 @@
+header {
+  width: 100%;
+  height: 10px;
+  background-color: lightpink;
+  position: fixed;
+  padding-top: 10px;
+  padding-bottom: 0px;
+  margin-bottom: 20px;
+  border-bottom: 1px solid wheat;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.burger div {
+  display: flex;
+  width: 25px;
+  height: 3px;
+  background-color: gray;
+  margin: 3px;
+}
+
+footer {
+  display: flex;
+  justify-content: center;
+  border-top: 1px solid beige;
+}
+
+.bebe_wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.pot_wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.shadow_wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
```

여기 부분들은 css 코드가 동일하니까 하나의 클래스명을 사용해서 재사용하면 좋겠죠 ?
각 아이템을 감싸는 div의 스타일링이니까

```suggestion
.item__wrapper {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
}
```
로 작성해서 재사용하면 더 깔끔해질거에요 !